src/main/groovy/nl/thehyve/gb/backend/user/User.groovy

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2019  The Hyve B.V.
+ *  This file is distributed under the GNU Affero General Public License
+ *  (see accompanying file LICENSE).
+ */
+
+package nl.thehyve.gb.backend.user
+
+import groovy.transform.CompileStatic
+import groovy.transform.Immutable
+
+@CompileStatic
+@Immutable
+class User {
```

Comment:
why do we need custom user been? Maybe it again could be solved by https://jira.thehyve.nl/browse/TMT-261